torch/csrc/autograd/functions/accumulate_grad.h

```diff
@@ -130,9 +131,9 @@ struct TORCH_API AccumulateGrad : public Node {
           new_grad._values().is_contiguous() &&
           // Use count for indices and values should always be <=1 since the
           // SparseTensor should be the only one holding a reference to these.
-          new_grad._indices().use_count() <= 1 &&
-          new_grad._values().use_count() <= 1 &&
-          new_grad.use_count() <= num_expected_refs) {
+          at::caching::adjusted_use_count(new_grad._indices()) <= 1 &&
```

Comment:
can sparse tensor indices and values ever be added to cache? I though cudagraph trees would require special handling for sparse tensors for this to happen, and right now trees just won't handle sparse tensors correctly?

Reply:
Yea, good point!